rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/evaluation/MUTEExamplesTest.kt

```diff
@@ -20,13 +20,8 @@ class MUTEExamplesTest {
     }
 
     @Test @Category(PerformanceTest::class)
-    fun executeMUTE10_04_A_perf_strings() {
-        assertEquals(listOf("001.1_d01.1_A01.1_c01.1_B01.1_b01.1_C01.1_901.1_101."), outputOf("performance/MUTE10_04_A"))
-    }
-
-    @Test @Category(PerformanceTest::class)
```

Comment:
Why this test has been removed?

Reply:
This test has been renamed, not removed.
MUTE10_04_A is not a valid source name (longer than 10 char) so I renamed it in MUTE10_04A.

Reply:
In this case the modification is so small that I thing it's worth it, anyway I think that, in the interpreter world, we could accept program names with longer names, especially if they are test programs